modular_skyrat/modules/customization/modules/language/piratespeak.dm

```diff
@@ -0,0 +1,6 @@
+/datum/language/piratespeak
+	secret = TRUE
+	desc = "The language of space pirates. Sounds similar to gutter, though intentionally filled with odd slang to make deciphering hard."
+	syllables = list(
+		"yaye", "alya", "chirka", "tikat", "bazar", "oi", "yo", "meit", "grok", "ken", "berk", "cohon", "pendeo", "mark"
```

Comment:
That looks incredibly small in terms of amount of different syllables.

Reply:
Yea, I'll add more syllables when I get around to it tomorrow!